cpp/log_rotate.cc

```diff
@@ -117,6 +127,12 @@ int Main(int argc, char *argv[]) {
                                                FileUtil::GroupnameFromGID(gid));
                     if (SELinuxUtil::IsEnabled())
                         SELinuxUtil::FileContext::ApplyChanges(filename);
+
+                    const pid_t our_pid(::getpid());
+                    for (const auto pid : pids) {
+                        if (pid != our_pid)
```

Comment:
GetProcessIdsForPath() sould take an additional bool to exclude the current pid, then we wouldn't have to test for our_pid here explicitly